code/game/objects/items/devices/dummy_tablet.dm

```diff
@@ -20,6 +27,12 @@
 	interact(user)
 
 /obj/item/device/professor_dummy_tablet/interact(mob/user as mob)
+	if (linked_dummy == null)
```

Comment:
```suggestion
	if (!linked_dummy)
```
alternatively, you can is `isnull(linked_dummy)`, but just checking for falsiness is fine